src/components/shared/NumberInput/NumberInput.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+
+import NumberInputContainer from './NumberInput.container';
+import Fab from 'components/shared/Fab';
+
+/**
+ * This component is a
+ * Number Input.
+ *
+ * @author jeremy
+ *
+ * Created: January 11, 2019
+ **/
+
+interface Props {
+  value: number;
+  max: number;
+  min: number;
+  onChange: (value: number) => void;
+  step: number;
+
+  end?: string;
+  font?: string;
+  prefixColor?: string;
+  prefixSize?: string;
```

Comment:
i think this, suffixSize, and svgSize arent being implemented atm?

Reply:
removed ty